ner.py

```diff
@@ -0,0 +1,46 @@
+from transformers import pipeline, AutoTokenizer
+
+# Load the local model and set up the pipeline
+model_name = "Gladiator/microsoft-deberta-v3-large_ner_conll2003"
+pipe = pipeline("token-classification", model=model_name)
+tokenizer = AutoTokenizer.from_pretrained(model_name)
+
+def ner(text):
+    results = pipe(text)
```

Comment:
**Suggestion:** The 'pipe' and 'tokenizer' variables are currently global variables, which could lead to potential issues if this script is imported as a module. It would be better to move them inside the 'ner' function to limit their scope.
```suggestion
def ner(text):

    pipe = pipeline("token-classification", model=model_name)
    tokenizer = AutoTokenizer.from_pretrained(model_name)

    results = pipe(text)
```